extensions/hooks/copy_pdbs_to_package.py

```diff
@@ -0,0 +1,30 @@
+from conan.tools.files import copy
+import glob
+import json
+import os
+import re
+
+
+def post_package(conanfile):
+    # Find dumpbin path
+    conanfile.run(
+        r'"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" -find "**\dumpbin.exe" -format json > out.json')
+    with open("out.json") as f:
+        dumpbin_path = json.load(f)[0]
+
+    # Find all dll paths in the package folder
+    search_package_dll = os.path.join(conanfile.package_folder, "**/*.dll")
+    package_dll = glob.glob(search_package_dll, recursive=True)
+    for dll_path in package_dll:
+        # Use dumpbin to get the pdb path from each dll
+        conanfile.run(rf'"{dumpbin_path}" /PDBPATH {dll_path} > out.txt')
+        with open("out.txt") as f:
+            dumpbin = f.read()
```

Comment:
```suggestion
        dumpbin_out = load(self, "out.txt")
```

Reply:
As I mentioned above, use the `stdout=` approach to load and parse the output, rather than relying on file redirections